src/pages/hegel/reference/becoming/sublation.mdx

```diff
@@ -0,0 +1,205 @@
+would be no mediation where any "pointing" would take place).
+
+To be something sublated, by contrast, entails a binding between its immediate
+state and its mediated state, or between its absolute- and relative scope.
+Diaception is the grasping of this binding, or holding in view both the absolute
+and relative scope of a category.
+
+In the context of `existence`, for example, where `being` and `nothing` are
+understood to be in a unity, `being` has its immediate state negated&mdash;there
+is no `pure being`&mdash;but in being _negated_ it is no less kept. Sublation
+entails diaception and thus implies development of a category.
+
+#### Sublation and Truth (Niklas)
+
+The development of a category in sublation is what Hegel refers to when he says
+that something has become understood _in its truth_. A category or thought
+underwent an internal dialectical transition through which it _turned out to be_
+something else&mdash;not something _completely_ different (the `being` of
+`existence` still shares something with `pure being`, its ancestor, so to
+speak), but also not something _entirely_ identical.
+
+The new sublated state refers back to its absolute being, and refers to it as
+negated, whereby it is simultaneously retained. It is this intense binding that
+grants something to be seen _in its truth_: a primordial state that was
+internally incoherent whose incoherence reforged it to this current state.
+
+In this way, _truth_, in light of Hegel's thinking, is never something immediate
+but the result of a development. It is a context made up of sublated moments,
+and whose conception understands that these moments were at one point totally
+unbounded, in their boundlessness utterly failed, and found themselves renewed
+in this multi-dimensional sphere as self-bounded moments.
+
+## Further Commentary
+
+### Houlgate
+
+Stephen Houlgate reads sublation as fusing two processes into one. The first
+process involves negating something while the second process involves preserving
+something. But what is specifically negated is the "simple _immediacy_ &ndash;
+or purity &ndash; of something" (Houlgate 2022, 153). For that reason, since
+what is sublated is no longer purely and immediately itself, Houlgate writes
+that "it must be thought together with its negation" (Houlgate 2022, 153). In
+this respect, it matches what Hegel writes: "something is sublated only in so
+far as it has entered into unity with its opposite" (Hegel 2010, 82/21.95).
+
+Houlgate points out that while `being` and `nothing` are sublated in `becoming`,
+this sublation is strictly incomplete. The sublation here is only partial
+because "each vanishes into its _pure other_, and their immediacy, or purity, is
+thus not thoroughly negated" (Houlgate 2022, 153). Viewed differently, the
+immediacy of each vanishes so completely that there is no remnant which could be
+considered a moment. By contrast, in `determinate being` (`existence`) the
```

Comment:
if there is such a vanishing, then can it really be said to be sublated?

Reply:
Well, no, and that's the point here, isn't it? You don't get sublation out of `being` and `nothing`, but only at the level of `becoming`.